src/components/range/range.ts

```diff
@@ -4,6 +4,7 @@ import {NG_VALUE_ACCESSOR} from '@angular/common';
 import {Form} from '../../util/form';
 import {isTrueProperty, isNumber, isString, isPresent, clamp} from '../../util/util';
 import {Item} from '../item/item';
+import {UIEventManager} from '../../util/ui-event-manager';
```

Comment:
This is cool and we should use UIEventManager here, and for all the similar use-cases, but can we keep this change separate from this PR?